backend/job_descriptions/router.py

```diff
@@ -13,15 +16,22 @@
 )
 
 
-@router.post("/")
+@router.post("/", response_model=JobDescriptionResponse)  # noqa: FAST001
```

Comment:
The "response_model" parameter is redundant when the return type is already annotated. FastAPI infers it automatically. Remove the parameter and the # noqa: FAST001 comment to let the linter catch these in the future.


```suggestion
@router.post("/")
```